library/network_connections.py

```diff
@@ -1954,6 +1926,14 @@ def __init__(self, **kwargs):
         self._nmutil = None
         self.validate_one_type = ArgValidator_ListConnections.VALIDATE_ONE_MODE_NM
         self._checkpoint = None
+        # pylint: disable=import-error, no-name-in-module
+        from ansible.module_utils.network_lsr.nm import (  # noqa: E501
```

Comment:
@nhosoi does the role2collection script handle this case?

Reply:
Once this [PR](https://github.com/linux-system-roles/auto-maintenance/pull/26) is merged, yes.